common/src/models/mnv2/mnv2.cc

```diff
@@ -64,22 +68,59 @@ static void do_classify_0() {
   tflite_set_input_unsigned(golden_tests[0].data);
   int32_t result = mnv2_classify();
   printf("Result is %ld\n", result);
+
+#ifdef CSR_VIDEO_FRAMEBUFFER_BASE
+  char msg_buff[256] = { 0 };
+
+  snprintf(msg_buff, sizeof(msg_buff), "Result is %ld", result);
+  fb_clear();
+  fb_draw_string(0,  10, 0x007FFF00, "Run test 0");
+  fb_draw_buffer(0,  50, 160, 160, (const uint8_t *)golden_tests[0].data, 3);
+  fb_draw_string(0, 220, 0x007FFF00, (const char *)msg_buff);
+  flush_cpu_dcache();
+  flush_l2_cache();
+#endif
 }
 
 static void do_classify_1() {
   tflite_set_input_unsigned(golden_tests[1].data);
   int32_t result = mnv2_classify();
   printf("Result is %ld\n", result);
+
+#ifdef CSR_VIDEO_FRAMEBUFFER_BASE
+  char msg_buff[256] = { 0 };
+
+  snprintf(msg_buff, sizeof(msg_buff), "Result is %ld", result);
+  fb_clear();
+  fb_draw_string(0,  10, 0x007FFF00, "Run test 1");
+  fb_draw_buffer(0,  50, 160, 160, (const uint8_t *)golden_tests[1].data, 3);
+  fb_draw_string(0, 220, 0x007FFF00, (const char *)msg_buff);
+  flush_cpu_dcache();
+  flush_l2_cache();
+#endif
 }
 
 static void do_classify_special() {
   tflite_set_input_unsigned(input_00001_18027);
   int32_t result = mnv2_classify();
   printf("Result is %ld\n", result);
+
+#ifdef CSR_VIDEO_FRAMEBUFFER_BASE
+  char msg_buff[256] = { 0 };
+
+  snprintf(msg_buff, sizeof(msg_buff), "Result is %ld", result);
+  fb_clear();
+  fb_draw_string(0, 10, 0x007FFF00, "Run special test");
+  fb_draw_buffer(0, 50, 160, 160, (const uint8_t *)input_00001_18027, 3);
+  fb_draw_string(0, 220, 0x007FFF00, (const char *)msg_buff);
+  flush_cpu_dcache();
+  flush_l2_cache();
+#endif
 }
 
 static void do_golden_tests() {
   bool failed = false;
+  char msg_buff[256] = { 0 };
```

Comment:
This needs to be guarded or it causes an error when CSR_VIDEO_FRAMEBUFFER_BASE is not defined.  Also in pdti8.cc.

Reply:
Hi, Tim:

Yes. Many thanks. I had make the change and re push into PR.

BR, Akio